src/main/groovy/de/triplet/gradle/play/PlayPublishListingTask.groovy

```diff
@@ -41,7 +41,7 @@ class PlayPublishListingTask extends PlayPublishTask {
         super.publish()
 
         // Matches if locale have the correct naming e.g. en-US for play store
-        inputFolder.eachDirMatch(matcher) { dir ->
+        if (inputFolder.exists()) inputFolder.eachDirMatch(matcher) { dir ->
```

Comment:
Yes, it is better to avoid executing any code when the inputFolder doesn't exist.